beacon_node/beacon_chain/src/block_verification.rs

```diff
@@ -605,7 +616,7 @@ pub fn signature_verify_chain_segment<T: BeaconChainTypes>(
         .collect::<Vec<_>>();
 
     // verify signatures
-    let pubkey_cache = get_validator_pubkey_cache(chain)?;
+    let pubkey_cache = get_validator_pubkey_cache(chain).map_err(BlockError::from)?;
```

Comment:
Doesn't look like we need this with the auto conversion?

```suggestion
    let pubkey_cache = get_validator_pubkey_cache(chain)?;
```